src/circuit/gadget/ecc/chip.rs

```diff
@@ -9,6 +9,7 @@ use halo2::{
     poly::Rotation,
 };
 
+mod double;
```

Comment:
If we don't use `double` in the actual circuit, remove this instruction from the gadget.